### ISSUE TITLE
Different kind of interaction for continuation of stack play (instants, abilities, etc) and phase change.

### ISSUE THREAD

**Issue by ShadowSFX** [magefree/mage#3785]
I've lost the count of how many times i've passed through a phase by accident due to lag. It'd would be great to have a different button / interaction to pass phases and to play cards as interrupts. Plus, it'd be nice to have a sound play each time a phase changes. This issue makes the interaction with the game extremely annoying to follow up unless you are completely used to it and it is still an issue if you have lag because you can very easily pass phases often by mistake.

**Comment by Zerrisx** [comment]
I... think this is already handled, between the F-buttons and phase-stops?  It's not necessarily easy / intuitive, but I'm not sure what this issue wants changed.  If you could clarify what you're asking for, I'd be happy to reopen this.

**Comment by ShadowSFX** [comment]
Yes, for example, the main problem is that the same label/same button is used for either continuing with the stack chain and to go to next phase. It'd be a lot more intuitive to have a specific button, the most left or most right from the "done" and "undo" buttons than having a "done" button for absolutely everything. I love the simplicity but in this specific case, it's rather complicated. I've been using xmage for years now and i still make the same mistake, i've brought a few people to play with me and they do the same thing, and the only choice is to rollback to start of turn but that's rather annoying as a player, specially if the turn has lasted quite a few minutes. Also, it'd be AMAZING! if we could choose to have the phase icons indicating which face we are at the middle horizontal or where it currently is. (not sure if it's already possible, i've searched for quite a while and i couldn't find a way to do it.

**Comment by Zerrisx** [comment]
Okay, cool - that's a concrete request, and one I don't know that we have immediate documented elsewhere.  Also one I fully support; that would be handy. Reopening.

**Comment by ShadowSFX** [comment]
Thanks! i'm super glad, i'll be looking forward to it :)

**Comment by JayDi85** [comment]
There are possible to add confirm dialog to change phase on non used action. See example with mana pool dialog (if you have mana and want to go next then it will ask you about it). Same for other things too:
1. You have attackers to attack;
2. You have defenders to defend;
3. You have mana in mana pool;
4. You have activated abilities to use (or only cast abilities);
5. Confirm dialog with style like
* confirm
* confirm and don't ask until end turn
* cancel

**Comment by ShadowSFX** [comment]
That could be too tedious. The current minimalistic functions are fine, it's just a few tweaks that can really help, like using a specific button to go to the next turn, being able to change the position/size of the phase icons and possibly a green glowy effect on the entire half of the screen of the player who has the turn instead of making the vertical rectangle of the player greenish. The current xmage is amazing, it really is just very few quality of life things that will make everyone happy
